packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2672,7 +2672,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     if (!_shouldCreateInputConnection) {
       _closeInputConnectionIfNeeded();
     } else if (oldWidget.readOnly && _hasFocus) {
-      _openInputConnection();
+      _openInputConnection(canTouchLayoutPhaseData: false);
```

Comment:
Or we could also just always wrap this in a post frame callback.  Though I feel like maybe _updateSizeAndTransform is the correct thing that should be aware of the build phase like that.

Reply:
That looks like another possibility, let me try it

Reply:
(implemented)